qa/workunits/rbd/rbd_groups.sh

```diff
@@ -217,23 +248,25 @@ echo "PASSED"
 echo "TEST: create remove snapshots of consistency group"
 image="test_image"
 group="test_consistency_group"
-snap="group_snap"
-new_snap="new_group_snap"
-sec_snap="group_snap2"
+snaps=("group_snap1" "group_snap2" "group_snap3" "group_snap4")
 create_image $image
 create_group $group
+create_snapshot $group ${snaps[0]}
```

Comment:
I'd pass an empty image name explicitly:
```suggestion
create_snapshot $group ${snaps[0]} ""
```